app/models/donation.rb

```diff
@@ -9,4 +9,8 @@ class Donation < ActiveRecord::Base
   validates :amount, presence: true, numericality: { greater_than: 0 }
 
   scope :search, -> (keyword) { joins(:donor).where("name ILIKE ?", "%#{keyword}%") }
+
+  def self.donations_for_current_year
```

Comment:
Let's do two things:

1. Change this into a scope, like the `.search` method.
2. Make it generalized so we can pass a year to it. 😀